tips/TIP-0046/tip-0046.md

```diff
@@ -0,0 +1,1001 @@
+            "mana": "2189"
+          },
+          {
+            "accountId": "0x7e0d0a5848362b23120f55115b096774036d7610137a631413221f5573344507",
+            "mana": "2285"
+          }
+        ],
+        "capabilities": "0x01",
+        "outputs": [
+          {
+            "type": 0,
+            "amount": "100000",
+            "mana": "0",
+            "unlockConditions": [
+              {
+                "type": 0,
+                "address": {
+                  "type": 0,
+                  "pubKeyHash": "0xed1484f4d1f7d8c037087fed661dd92faccae1eed3c01182d6fdd6828cea144a"
+                }
+              }
+            ],
+            "features": [
+              {
+                "type": 5,
+                "id": "0x086372557616532f714f104e5f44297b7a286d077956291a6d4f59081f484463712a64300c00",
+                "amount": "0x14be8149371263f4"
+              }
+            ]
+          },
+          {
+            "type": 1,
+            "amount": "100000",
+            "mana": "5000",
+            "accountId": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "foundryCounter": 0,
+            "unlockConditions": [
+              {
+                "type": 0,
+                "address": {
+                  "type": 0,
+                  "pubKeyHash": "0xed1484f4d1f7d8c037087fed661dd92faccae1eed3c01182d6fdd6828cea144a"
+                }
+              }
+            ],
+            "features": [
+              {
+                "type": 2,
+                "entries": {
+                  "hello": "0x776f726c64"
+                }
+              },
+              {
+                "type": 6,
+                "expirySlot": 4294967295,
+                "blockIssuerKeys": [
+                  {
+                    "type": 0,
+                    "pubKeyHash": "0x295409de79016133647d4078cb01618a4ba018eb74ff613138d8ff8dc05de73c"
+                  },
+                  {
+                    "type": 0,
+                    "pubKeyHash": "0x868f4c6ef7b5b1d55838cbfb8ae4f3a9776c53cdd3e3d33000094d72acab5a2f"
+                  }
+                ]
+              },
+              {
+                "type": 7,
+                "stakedAmount": "10000",
+                "fixedCost": "400",
+                "startEpoch": 0,
+                "endEpoch": 4294967295
+              }
+            ]
+          }
+        ]
+      },
+      "unlocks": [
+        {
+          "type": 0,
+          "signature": {
+            "type": 0,
+            "publicKey": "0x2daefbcbadd044da470acd2f7fcf6fcb04b873cc801e7ee408018e1dfa0257ac",
+            "signature": "0x5bb409d59e01d2ea9f1a1fb67feb681d0d3ecb05787cadad2f89fdf13ef7ff03ad5cebf28df5dddd8510992596d98b133f86e14f76824e6ccc369a8f5df44806"
+          }
+        },
+        {
+          "type": 1,
+          "reference": 0
+        }
+      ]
+    },
+    "maxBurnedMana": "864"
```

Comment:
Looks weird that it is set to same as for `Basic Block ID & Work Score (Tagged Data payload)`

Reply:
It was hardcoded before but I've set it to `minRMC * payloadWorkScore` now to give it a more realistic value.